centre-web/src/components/LaunchAppTile/BookmarkSection.tsx

```diff
@@ -71,22 +71,27 @@ export const BookmarkSection = ({ epicApp }: BookmarkSectionProps) => {
                   <CentreLink
                     key={bookmark.label}
                     onClick={() => window.open(bookmark.url, "_blank")}
+                    sx={{
+                      display: "flex",
+                      alignItems: "center",
+                      overflow: "hidden",
+                      textOverflow: "ellipsis",
+                      whiteSpace: "nowrap",
+                    }}
                   >
+                    {bookmark.label && bookmark.label.length > 0 && (
+                      <Circle sx={{ marginRight: "5px", fontSize: "7px", flexShrink: 0 }} />
+                    )}
```

Comment:
would this package help. It is used already in the app. Allows for deciding how many lines are wrapped before the ellipsis. If it doesn't help for this use case nevermind:

<img width="738" height="454" alt="Image" src="https://github.com/user-attachments/assets/cdbf431f-1b67-43ef-9f67-4c5c7d12eeef" />